src/common/data/stores/app/homebase/homebaseStore.ts

```diff
@@ -59,6 +60,16 @@ export const createHomeBaseStoreFunc = (
       const spaceConfig = JSON.parse(
         await get().account.decryptEncryptedSignedFile(fileData),
       ) as SpaceConfig;
+      const currentHomebase = get().homebase.homebaseConfig;
+      if (
+        spaceConfig.timestamp &&
+        currentHomebase &&
+        currentHomebase.timestamp &&
```

Comment:
Should also return here if local config has a timestamp and the fetched one doesn't